lib/vsc/modules/cache.py

```diff
@@ -76,24 +75,31 @@ def _cmp(self, other):
         try:
             return super(SoftwareVersion, self)._cmp(other)
         except Exception as e:
-            LOGGER.error("Failed to compare %s (%s) with other %s (%s): %s",
+            logging.exception("Failed to compare %s (%s) with other %s (%s): %s",
                          self, self.version, other, other.version, e)
             raise
 
 
+def log_and_raise(msg):
```

Comment:
Ideally, this is made available more generally (later on).

Reply:
ideally, that way of working goes away :smile: